packages/cli/src/keys/keys.ts

```diff
@@ -0,0 +1,38 @@
+import crypto from "crypto";
+import * as ed25519 from "@transmute/did-key-ed25519";
+import * as x25519 from "@transmute/did-key-x25519";
+import * as secp256k1 from "@transmute/did-key-secp256k1";
+import * as bls12381 from "@transmute/did-key-bls12381";
+import * as didKeyWeb from "@transmute/did-key-web-crypto";
+
+export const generateKeys = async (type: string): Promise<any[]> => {
+  const options = {
+    secureRandom: () => {
+      return crypto.randomBytes(32);
+    }
+  };
+  switch (type) {
+    case "ed25519": {
+      return [await ed25519.Ed25519KeyPair.generate(options)];
+    }
+    case "x25519": {
+      return [await x25519.X25519KeyPair.generate(options)];
+    }
+    case "bls12381": {
+      const keypairs = await bls12381.Bls12381KeyPairs.generate();
+      return [keypairs.g1KeyPair, keypairs.g2KeyPair];
+    }
+    case "p-256": {
```

Comment:
we might want to switch this to P-384 based on the SVIP Profile stuff, and yes, there are folks who only use P-384... in NZ...

Reply:
agreed - p384 can come later - basically stubbing in some stuff to match universal wallet, then adding credentials, then going back and expanding key types